### PR TITLE
Spotlight

### PR DESCRIPTION
**Pull request by dafnevk**
I prepared text for the [Carpentries Incubator Lesson Spotlight](https://docs.carpentries.org/topic_folders/lesson_development/spotlight.html), where this lesson could be featured. 

When you agree and the PR is approved or merged I will submit the answers to the form as well.

### CODE REVIEW

**Review by colinsauze — COMMENTED** [review]
see inline comment about target audience, otherwise it looks good to me.

**Comment by dsmits** [comment]
I'm a bit confused about the commits in this PR. Are wikfeldts commits supposed to be in this PR?

File: _meta/spotlight.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Introduction to Deep Learning
+
+## What is the URL of your lesson pages?
+https://carpentries-incubator.github.io/deep-learning-intro
+
+## What is the URL of your lesson GitHub repository?
+https://github.com/carpentries-incubator/deep-learning-intro
+
+## Who is the target audience of your lesson? (1-3 sentences)
+The main audience of this carpentry lesson is PhD students that have little to no experience with deep learning. In addition, we expect them to know basics of statistics and machine learning.
```
[the remark this answers is not in the document]
Comment (psteinb) [reply _meta/spotlight.md:22]: I personally wouldn't limit the target audience to academia at all.

```suggestion
The main audience of this carpentry lesson is considered to have an academic background at any level. More importantly, we expect them to know basics of statistics and machine learning to follow through with the material.
```

File: _meta/spotlight.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Introduction to Deep learning
+
+## What name(s) should be listed as author(s) of the Incubator Lesson Spotlight blog post?
+- Colin Sauze
+- Dafne van Kuppevelt
+- Djura Smits
+- Peter Steinbach
+
+## Please provide the GitHub username(s) of the author(s) who will review the final blog post before it is published.
+dafnevk
```
Comment (psteinb) [inline _meta/spotlight.md:10]: I am happy to help with this too.

**Comment by dafnevk** [comment]
> I'm a bit confused about the commits in this PR. Are wikfeldts commits supposed to be in this PR?

I'm sorry this is probably my fault, maybe I forked from his PR. Shouldn't matter in the end because we merged his PR with those commits so I think it's safe to only look at the `spotlight.md` file.

**Review by dsmits — APPROVED** [review]

**Pull request merged**